packages/components/pie-modal/src/index.ts

```diff
@@ -1,14 +1,23 @@
-import { LitElement, html } from 'lit'; // eslint-disable-line import/no-extraneous-dependencies
+import { LitElement, html, unsafeCSS } from 'lit'; // eslint-disable-line import/no-extraneous-dependencies
 import { RtlMixin } from '@justeattakeaway/pie-webc-core';
 
+import styles from './modal.scss?inline';
+
 export class PieModal extends RtlMixin(LitElement) {
     // eslint-disable-next-line class-methods-use-this
     render () {
         return html`
-            <div>This is the Pie Modal
-                <slot></slot>
-            </div>`;
+            <dialog class="o-modal" open>
+                <h3 class="o-modal-header">Modal header</h3>
+                <div class="o-modal-content-wrapper">
```

Comment:
Nit – for the naming, would be `contentWrapper` (as dashes show an extra level of DOM hierarchy)

```suggestion
                <div class="c-modal-contentWrapper">
```